tc/sgx/common/crypto/verify_certificate.cpp

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+#include <openssl/x509v3.h>
+#include <openssl/pem.h>
+
+#include "verify_certificate.h"
+#include "ias-certificates.h"
```

Comment:
remove ias-certificates.h dependency in crypto module

Reply:
Done